src/index.js

```diff
@@ -36,3 +36,6 @@ $('#splash__start--button--js').on('click', function() {
   domUpdates.startGame(game);
   game.createPlayers(playerOne, playerTwo, playerThree)
```

Comment:
This could get invoked in `game.startGame`, especially if it's a prerequisite for a game/Round to begin